tests/test_jwt.py

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/python2.4
-#
-# Copyright 2014 Google Inc. All rights reserved.
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Don't "update" copyright years in license headers. New files should always get the year of their introduction, but existing license headers should not be altered.